tests/test_cli.py

```diff
@@ -14,12 +14,12 @@
 
 _thisdir = os.path.abspath(os.path.dirname(__file__))
 
-InitArgs = collections.namedtuple(
+InitArgs = collections.namedtuple( # type: ignore
     "ArgsObject",
```

Comment:
I think you can resolve this by passing `"InitArgs"` as the first argument instead of `"ArgsObject"`, if I understood [this](https://docs.python.org/3/library/collections.html#collections.namedtuple) correctly. Makes no difference to the tests, at least.

Same for the one below, pass in "RegenerateArgs" instead.

Reply:
i was afraid this might break the test so i ignored it but it worked! updated it.